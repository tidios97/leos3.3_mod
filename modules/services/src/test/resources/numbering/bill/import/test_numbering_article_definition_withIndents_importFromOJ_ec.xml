<article xml:id="imp_art_d553e23657_MHFxJd">
    <num>Article 272</num>
    <heading>Definitions</heading>
    <paragraph>
        <subparagraph>
            <content>
                <p>Sub paragraph content</p>
            </content>
        </subparagraph>
        <list>
            <indent>
                <num></num>
                <alinea>
                    <content>
                        <p>General terms</p>
                    </content>
                </alinea>
                <list>
                    <point>
                        <num>(1)</num>
                        <content>
                            <p>content of (1)</p>
                        </content>
                    </point>
                </list>
            </indent>
            <indent>
                <num></num>
                <alinea>
                    <content>
                        <p>Transaction types</p>
                    </content>
                </alinea>
                <list>
                    <point>
                        <num>(2)</num>
                        <content>
                            <p>content of (2)</p>
                        </content>
                    </point>
                    <point>
                        <num>(3)</num>
                        <content>
                            <p>content of (3)</p>
                        </content>
                    </point>
                </list>
            </indent>
            <indent>
                <num></num>
                <alinea>
                    <content>
                        <p>Netting set, hedging sets, and related terms</p>
                    </content>
                </alinea>
                <list>
                    <point>
                        <num>(4)</num>
                        <alinea>
                            <content>
                                <p>content of (4)</p>
                            </content>
                        </alinea>
                        <alinea>
                            <content>
                                <p>first Alinea</p>
                            </content>
                        </alinea>
                        <alinea>
                            <content>
                                <p>second Alinea</p>
                            </content>
                        </alinea>
                    </point>
                </list>
            </indent>
        </list>
    </paragraph>
</article>