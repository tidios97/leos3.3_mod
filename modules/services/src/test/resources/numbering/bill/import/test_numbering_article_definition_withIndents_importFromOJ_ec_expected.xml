<article xml:id="imp_art_d553e23657_MHFxJd">
    <num>Article 1</num>
    <heading>Definitions</heading>
    <paragraph>
        <subparagraph>
            <content>
                <p>Sub paragraph content</p>
            </content>
        </subparagraph>
        <list>
            <indent>
                <num></num>
                <alinea>
                    <content>
                        <p>General terms</p>
                    </content>
                </alinea>
                <list>
                    <point>
                        <num>(1)</num>
                        <content>
                            <p>Point (a)</p>
                        </content>
                    </point>
                </list>
            </indent>
            <indent>
                <num></num>
                <alinea>
                    <content>
                        <p>Transaction types</p>
                    </content>
                </alinea>
                <list>
                    <point>
                        <num>(2)</num>
                        <content>
                            <p>Point (a)</p>
                        </content>
                    </point>
                    <point>
                        <num>(3)</num>
                        <content>
                            <p>Point (a)</p>
                        </content>
                    </point>
                </list>
            </indent>
            <indent>
                <num></num>
                <alinea>
                    <content>
                        <p>Netting set, hedging sets, and related terms</p>
                    </content>
                </alinea>
                <list>
                    <point>
                        <num>(4)</num>
                        <alinea>
                            <content>
                                <p>Point (a)</p>
                            </content>
                        </alinea>
                        <alinea>
                            <content>
                                <p>Alinea</p>
                            </content>
                        </alinea>
                        <alinea>
                            <content>
                                <p>Alinea</p>
                            </content>
                        </alinea>
                    </point>
                </list>
            </indent>
        </list>
    </paragraph>
</article>